app/controllers/api/v1/users_controller.rb

```diff
@@ -9,6 +9,14 @@ def index
     set_user
   end
 
+  # GET /api/v1/user
+  def user
+    render json: {
+      message: 'SUCCESS',
+      data: current_user,
+    }, status: :ok
+  end
+
```

Comment:
Just a note, we should keep in mind that we want to make sure there is no way this can catch an error in normal use case